app/models/articles.models.js

```diff
@@ -1,15 +1,30 @@
 const { db } = require('../../db/connection')
 
 exports.fetchArticles = (topic) => {
-    return db.query (   `SELECT
-                        articles.author, articles.title, articles.article_id, articles.topic,
-                        articles.created_at, articles.votes, articles.article_img_url,
-                        COUNT(comments.comment_id) AS comment_count
-                        FROM articles
-                        LEFT JOIN comments ON comments.article_id = articles.article_id
-                        GROUP BY articles.author, articles.title, articles.article_id, articles.topic,
-                        articles.created_at, articles.votes, articles.article_img_url
-                        ORDER BY articles.created_at DESC;`)
+    const validTopics = {
+        mitch: 'mitch',
+        cats: 'cats'
+    }
```

Comment:
I think hardcoding which topics can be accepted isn't quite the best idea - what if someone adds another topic?